### PR TITLE
refactor: swap frontend fee

### PR DESCRIPTION
**Pull request by stackchain**

### CODE REVIEW

**Review by michalszorad — APPROVED** [review]

File: packages/api/src/app/api/frontend-fees.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      if (!parsedResponse)
+        return Promise.reject(new Error('Invalid frontend fee response'))
+
+      return Promise.resolve(parsedResponse)
+    })
+  }
+
+const SwapAggregatorSchema = z.union([
+  z.literal('muesliswap'),
+  z.literal('dexhunter'),
```
Comment (SorinC6) [inline packages/api/src/app/api/frontend-fees.ts]: @stackchain, related to the `dexhunter` in which case will use it? as is the first time I have seen this mentioned. I thought we would be using only `muesliswap`

Comment (stackchain) [reply packages/api/src/app/api/frontend-fees.ts]: hey @SorinC6 no worries, it's not for now.

**Review by SorinC6 — COMMENTED** [review]
@stackchain shouldn't we add the frontend fee when creating the`createYoroiEntry` as well ?

**Comment by michalszorad** [comment]
@SorinC6 yes, I'm working that piece 😄

**Comment by stackchain** [comment]
> @stackchain shouldn't we add the frontend fee when creating the`createYoroiEntry` as well ?

If is gte 0 yap, @MichalSzorad is checking the lib side to see if an `Entry` with targets will work, we are also blocked by the wallet address from management to perform the deposits.

**Pull request merged**